<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OMSClass</class>
 <widget class="QMainWindow" name="OMSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Object Management System</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="imageView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>631</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectImg_Button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Image(s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processImg_Button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileName">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Source Image:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileName_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>140</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Processed Image:</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="imageView_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>611</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="back_Button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>610</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="next_Button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>610</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="OMS.qrc"/>
 </resources>
 <connections/>
</ui>
